examples/dell5k_edge_system_iotcc.py

```diff
@@ -69,7 +69,7 @@ def read_disk_usage_stats():
 
 
 def read_network_bytes_received():
-    return round(net_stat.rx_tx_bytes('eth0')[0], 2)
+    return round(net_stat.rx_tx_bytes('ens33')[0], 2)
```

Comment:
Thanks for pointing this out. We should surely take up the better approach but as this code is required urgently we should go ahead and push the code. Will open an issue for the same.